tkweb/apps/drinks/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+import subprocess
+from django.core.files import File
+
+
+class Drink(models.Model):
+    name = models.CharField(max_length=30)
+    serving = models.CharField(max_length=50)
+    soda = models.CharField(max_length=50)
+    price = models.IntegerField()
+
+    def __str__(self):
+        return self.name
+
+class Sprut(models.Model):
+    drink = models.ForeignKey(Drink, on_delete=models.CASCADE) 
+    name = models.CharField(max_length=30)
+    amount = models.IntegerField()
+
+    def __str__(self):
+        return self.name
+
+class Barcard(models.Model):
+    name = models.CharField(max_length=30)
+    drinks = models.ManyToManyField(Drink)
+    barcardFile = models.FileField(blank=True, upload_to='barcard') 
+    mixingFile = models.FileField(blank=True, upload_to='mixing') 
+    
+    def __str__(self):
+        return self.name
+
+    def generateFiles(self):
+        filePath = 'tkweb/apps/drinks/drinkskort/drinks.txt'
+        with open(filePath, 'w') as f:
+            for drink in self.drinks.all():
+                f.write('= '+drink.name+'\n')
+                for sprut in drink.sprut_set.all():
+                    f.write('- '+str(sprut.amount)+' cl - '+sprut.name+'\n')
+                f.write('-- '+drink.soda+'\n')
+                f.write('! '+drink.serving+'\n')
+                f.write('$ '+str(drink.price)+'\n')
+        bashCommand = 'make -C tkweb/apps/drinks/drinkskort/'
+        subprocess.call(bashCommand, shell=True)
```

Comment:
Jeg er i tvivl om hvorvidt det vil virke på serveren når det kører i Apache+mod_wsgi i stedet for `manage.py runserver`. For en sikkerheds skyld bør det rettes så det ikke afhænger af hvad den nuværende mappe er for Python:

```
subprocess.call("make", cwd=os.path.dirname(__file__) + "/drinkskort")
```